### ISSUE TITLE
Please publish the ST_AsMVT new signature to npm

### ISSUE THREAD

**Issue by luizotcarvalho** [Stezii/tilestrata-postgismvt#5]
I'm trying to use this library with npm but the signature is wrong, can you update the npm package? 

thanks.

**Comment by lamuertepeluda** [comment]
See https://github.com/Geodan/postgis-mvt-server/commit/2e87d58c3141b1f6ad66acec59670eb9ad8ce695

Now q is the first parameter

**Comment by luizotcarvalho** [comment]
@lamuertepeluda yes, I aware thats was fixed, but in npm package its still with the old signature. Im asking to @Stezii publish this new version there.

https://www.npmjs.com/package/tilestrata-postgismvt

**Comment by luizotcarvalho** [comment]
For now, I'm installing direct from github.

Heres my package.json
```
{
 "devDependencies": {
    "tilestrata": "^2.1.2",
    "tilestrata-postgismvt": "github:Stezii/tilestrata-postgismvt"
  }
}
```
